### PR TITLE
Fix GitHub workflow badges

### LINKED ISSUE #5461: Github CI build badge is broken on default v1.x branch
### Section/Content To Improve

In the README.md, on the default branch v1.x, in the badges, the 'CI' badge shows with a link to a shields issue rather than an actual CI status.

### Suggested Improvement

Hi,

I saw that the 'CI' badge was broken for axios on the 'default' v1.x branch - it looks like it's #5046 that breaks it on v1.x - v1 and main are both fine.

Update the url to remove the shields issue link and link to the actual CI status.

Currently badge looks like: 

Screenshot 2023-01-11 at 11 27 33

The shields link explains the issue, but in summary, these badges were changed in a breaking fashion that means they need a url update, so the badges were changed to just link to the github issue rather than showing the data.

Updating the url from:

https://img.shields.io/github/workflow/status/axios/axios/ci?label=CI&logo=github&style=flat-square

to

https://img.shields.io/github/actions/workflow/status/axios/axios/ci.yml?branch=v1.x&label=CI&logo=github&style=flat-square

solves this (although note that I've specified the branch as v1.x - unclear to me if this is correct, or if it should be main, or something else.

### Relevant File(s)

```bash
README.md
```

### PR DESCRIPTION
**Pull request by EFord36**
See the linked GitHub issue from the previous version of the badge: https://github.com/badges/shields/issues/8671

In summary, these badges were changed in a breaking fashion such that the badges were just linking to the github issue rather than showing the data. Updating the url resolves this.

## Confirmation needed from a maintainer: what branch to use for build badge?

Note that it's recommended (in the above link) to specify a branch - I've specified v1.x as this is the default branch: please let me know if I should be using `main` instead, or something else. I'm not familiar with axios' development branching approach and couldn't find anything from a quick search that showed me what was correct here.

closes #5461

### CODE REVIEW

**Comment by EFord36** [comment]
force-push was to make the commit message follow conventional commits - sorry for missing that the first time!

**Review by jasonsaayman — APPROVED** [review]

**Comment by EFord36** [comment]
Thanks for approving! Let me know if there's anything I need to do before merge (but no rush, just wanted to check you aren't waiting on me to do something). Looks like I can't run workflows or merge myself.

**Comment by DigitalBrainJS** [comment]
Thank you for your contribution! 👍

**Pull request merged**
